doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -0,0 +1,864 @@
+
+See the changelog for each library in the :doc:`nfxlib documentation <nrfxlib:README>` for additional information.
+
+* Added:
+
+  * :ref:`nrfxlib:nrf_dm` and corresponding overview, usage, changelog, and API documentation.
+
+* Updated:
+
+  * :ref:`crypto`:
+
+    * Added PSA crypto drivers for CryptoCell and Oberon.
+    * Updated:
+
+      * :ref:`nrfxlib:nrf_cc3xx_platform_readme` to v0.9.13.
+      * :ref:`nrfxlib:nrf_cc3xx_mbedcrypto_readme` to v0.9.13.
+      * :ref:`nrfxlib:nrf_oberon_readme` to v3.0.10.
+
+    * Deprecated the ability of Mbed TLS API to enable multiple cryptographic backends in a build, such as the Oberon accelerated library or CryptoCell hardware accelerator. This applies only to the APIs with the ``mbedtls_`` prefix.
+
+  * :ref:`softdevice_controller`:
+
+    * Updated:
+
+      * :ref:`softdevice_controller_scheduling` by updating :ref:`scheduling_priorities_table` and adding a section on Timing when synchronized to a periodic advertiser.
+
+  * :ref:`nrf_modem`:
+
+    * Updated:
+
+      * :ref:`nrfxlib:nrf_modem` to v1.5.1.
+      * Architecture diagram in :ref:`nrfxlib:architecture`.
+      * :ref:`nrfxlib:nrf_modem_delta_dfu`.
+
+    * Removed:
+
+      * Overview diagram in :ref:`nrfxlib:nrf_modem`.
+      * AT socket page.
+      * Extensions page.
+
+  * :ref:`nrf_security`:
+
+    * Added experimental support for PSA crypto drivers in :ref:`nrf_security`.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+Unity
+-----
+
+* Fixed a bug that resulted in the mocks for some functions not having the ``__wrap_`` prefix. This happened for functions declared with whitespaces between identifier and parameter list.
+
+HID Configurator for nRF Desktop
+--------------------------------
+
+* Updated:
+
+  * The script to recognize the bootloader variant as a DFU module variant for the configuration channel communication. The new implementation is backward compatible - the new version of the script checks for module name and acts accordingly. See the Performing DFU section in :ref:`nrf_desktop_config_channel_script` documentation.
+
+  * Documentation on setting up the dependencies for Debian/Ubuntu/Linux Mint operating system.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``680ed07``, plus some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* |no_changes_yet_note|
+
+Mcumgr
+======
+
+The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``657deb65``.
+
+The following list summarizes the most important changes inherited from upstream mcumgr:
+
+* |no_changes_yet_note|
+
+Zephyr
```

Comment:
@carlescufi , could you check this section?

Reply:
@SebastianBoe  and @mbolivar-nordic please go through this section.

Reply:
done